djangoproject/templates/releases/download.html

```diff
@@ -162,6 +156,12 @@ <h2 id="unsupported-versions">Unsupported previous releases</h2>
       <td>August 3, 2022</td>
       <td>April 1, 2023</td>
     </tr>
+    <tr>
+      <td>3.2 LTS</td>
+      <td>{% get_latest_micro_release '3.2' %}</td>
+      <td>December 7, 2021</td>
+      <td>April 3, 2024</td>
```

Comment:
We didn't release a new version so I'd use `April 1`.
```suggestion
      <td>April 1, 2024</td>
```

Reply:
As log as it's not read as an April's fool... :smile: